client/blocks/app-banner/utils.js

```diff
@@ -55,24 +60,33 @@ export function getCurrentSection( currentSection, isNotesOpen ) {
 	return null;
 }
 
-export function getNewDismissTimes( dismissedSection, currentDismissTimes ) {
+function getDismissTimes( isControl ) {
 	const currentTime = Date.now();
-	const aWeekFromNow = currentTime + ONE_WEEK_IN_MILLISECONDS;
-	const aMonthFromNow = currentTime + ONE_MONTH_IN_MILLISECONDS;
+	const currentSection = isControl ? ONE_MONTH_IN_MILLISECONDS : TWO_WEEKS_IN_MILLISECONDS;
+	const otherSections = isControl ? ONE_WEEK_IN_MILLISECONDS : ONE_DAY_IN_MILLISECONDS;
+
+	return {
+		current: currentTime + currentSection,
```

Comment:
instead of `current` and `other` I think something like `longer_duration` and `shorter_duration` would work better here.

Reply:
Done in: 91b2eed56e78ca6ee80ab03f8cdab03c352719cd